library/src/main/java/com/novoda/viewpageradapter/ViewPagerAdapter.java

```diff
@@ -47,24 +58,34 @@ public V instantiateItem(ViewGroup container, int position) {
     /**
      * Bind the view to the item at the given position with view state.
      *
-     * @param view      the view to bind
+     * @param view      the page view to bind
      * @param position  the position of the data set that is to be represented by this view
      * @param viewState the state of the view
      */
-    protected void bindView(V view, int position, @Nullable SparseArray<Parcelable> viewState) {
+    private void bindView(V view, int position, @Nullable SparseArray<Parcelable> viewState) {
```

Comment:
this is a breaking API change. Verified with @ouchadam that it's compatible with the usage in the internal project, now that we expose `restoreHierarchyState`